src/ScriptExecutor.php

```diff
@@ -86,7 +86,7 @@ private function expandCmd(string $type, string $cmd)
             case 'script':
                 return $cmd;
             default:
-                throw new \InvalidArgumentException(sprintf('Command type "%s" is not valid.', $type));
+                throw new \InvalidArgumentException(sprintf('Invalid symfony/flex auto-script in composer.json: "%s" is not a valid type of command.', $type));
```

Comment:
would suggest the following:

`type for command "%s", $type, $cmd`

as the type in my case is 0, 1, 2, 3

Reply:
Are you sure? The type is the right side, so it's the string here to me. I also ran out he code to reproduce your situation and the error was fine.

Reply:
@nicolas-grekas sorry you are correct!